### ISSUE TITLE
support uniswap v3 pricefeeds

### ISSUE THREAD

**Issue by hrishikeshio** [razor-network/datasources#37]
https://etherscan.io/address/0xB210CE856631EeEB767eFa666EC7C1C57738d438#readContract#F7

0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48
100
0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48
0x50de6856358cc35f3a9a57eaaa34bd4cb707d2cd
0x45CDEc62E3e8866C917F8AB76F7B1D45DDE6673E
2000

**Comment by hrishikeshio** [comment]
Any update @rajkharvar ?
Its pending since long time and it is a high priority. Please complete asap.

**Comment by rajkharvar** [comment]
Oops, Sorry I missed this. Will look into this now.

**Comment by rajkharvar** [comment]
@hrishikeshio It's not possible to query blockchain using simple HTTP GET request. If we want to support uniswap pricefeed though we might need a proxy server which fetches data from blockchain and serve it through GET request.

**Comment by hrishikeshio** [comment]
I had explained how to do it using etherscan

**Comment by rajkharvar** [comment]
Yes, through POST request it's possible but through GET request it's not possible.

**Comment by rajkharvar** [comment]
@hrishikeshio Tried to fetch blocknumber using GET request from alchemy and infura node, but it's not supported.

I got this error.

Alchemy:

```{"message":"Method Not Allowed","logref":null,"path":null,"_links":{"self":{"href":"/v2/Secb_SIZP3BfiGJKmVCDjxsbTf","templated":false,"profile":null,"deprecation":null,"title":null,"hreflang":null,"type":null,"name":null}},"_embedded":{"errors":[{"message":"Method [GET] not allowed for URI [/v2/Secb_SIZP3BfiGJKmVCDjxsbTf]. Allowed methods: [POST]","logref":null,"path":null,"_links":{},"_embedded":{}}]}}```

Infura:
```404 page not found```

**Comment by SkandaBhat** [comment]
Compatibility for this has been achieved in https://github.com/razor-network/oracle-node/pull/1020. This will be released in v1.0.6.
